client/signup/steps/domains/index.jsx

```diff
@@ -65,6 +65,26 @@ class DomainsStep extends React.Component {
 	constructor( props ) {
 		super( props );
 
+		const { flowName, signupDependencies } = props;
+		const importUrl = get( signupDependencies, 'importUrl' );
+
+		if ( flowName === 'from-site' && importUrl ) {
+			this.skipRender = true;
+
+			SignupActions.submitSignupStep(
+				Object.assign( {
```

Comment:
What's the benefit of `Object.assign` here?

Reply:
Unsure, but other usage has it.  Assume intention to pass by value.

Reply:
I might be wrong, but I think a plain object would do the same job no?

Reply:
Would a plain object mean that the values of the object (like importUrl) were passed by reference, not value? (I'm not sure...)

Reply:
If the object being passed had other objects as keys, they would be passed by reference like so:

```
const testObj = { a: { v: 'initial' } };
const testFn = obj => obj.a.v = 'mutated';
testFn( testObj );
testObj.a.v === 'mutated';
```

...but the passed in object here has no other reference & only has strings as values (which always pass by value in JS), so it's safe to move to a regular object syntax.  Will do.

So, ya